share/picongpu/examples/FoilLCT/etc/picongpu/4_isaac.cfg

```diff
@@ -0,0 +1,85 @@
+# Copyright 2017 Axel Huebl
+#
+# This file is part of PIConGPU.
+#
+# PIConGPU is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PIConGPU is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PIConGPU.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+
+##
+## This configuration file is used by PIConGPU's TBG tool to create a
+## batch script for PIConGPU runs. For a detailed description of PIConGPU
+## configuration files including all available variables, see
+##
+##                      docs/TBG_macros.cfg
+##
+
+
+#################################
+## Section: Required Variables ##
+#################################
+
+TBG_wallTime="23:59:00"
+
+TBG_gpu_x=2
+TBG_gpu_y=2
+TBG_gpu_z=1
+
+TBG_gridSize="-g 192 192 192"
+TBG_steps="-s 6000"
+
+TBG_periodic="--periodic 0 0 0"
+
+#################################
+## Section: Optional Variables ##
+#################################
+
+TBG_isaac="--isaac.period 1 --isaac.name foil --isaac.url hypnos5 --isaac.quality 99"
```

Comment:
unrecognised option '--isaac.period'

Reply:
probably because I forgot to change the 2D foil example into a 3D setup, right @theZiz ?

Reply:
Probably: https://github.com/ComputationalRadiationPhysics/picongpu/blob/dev/include/picongpu/plugins/PluginController.hpp#L60